nss-cohort-36/c36-class-website
# deleted 2 pictures

# Description
the pictures are in

## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality to see pics)

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-photos`
4. [Enter specific testing instructions here to approve]
# Checklist:
